Client/Frontend/Strings.swift

```diff
@@ -56,7 +56,28 @@ private func MZLocalizedString(
 // MARK: - Alerts
 extension String {
     public struct Alerts {
-
+        public struct RestoreTabs {
+            public static let Title = MZLocalizedString(
+                "Alerts.RestoreTabs.Title.v109",
+                tableName: "Alerts",
+                value: "Firefox crashed. Restore your tabs?",
+                comment: "The title of the restore tabs pop-up alert.")
+            public static let Message = MZLocalizedString(
+                "Alerts.RestoreTabs.Message.v109",
+                tableName: "Alerts",
+                value: "Sorry about that. Restore tabs to pick up where you left off.",
+                comment: "The body of the restore tabs pop-up alert.")
+            public static let ButtonNo = MZLocalizedString(
+                "Alerts.RestoreTabs.Button.No.v109",
+                tableName: "Alerts",
+                value: "No",
+                comment: "The title for the negative action of the restore tabs pop-up alert")
```

Comment:
For a "no" string like this, I would add more context in the comment.
Something like:
"The title for the negative action of the restore tabs pop-up alert. This alert comes up when etc etc etc, and will reject the action of restoring tabs"